### PR TITLE
Add CSS for layout and buttons

### PR DESCRIPTION
**Pull request by aprilsourz**

### CODE REVIEW

File: app/templates/components/my-application.hbs [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-        {{#paper-button}}{{paper-icon 'home'}}Home{{/paper-button}}
-      {{/link-to}}
+    <h1 class="text-center header-title">Free Lunch</h1>
+{{/paper-toolbar}}
+
+<div class="toolbar-two">
+  {{#if isAuthenticated}}
 
+    {{#if isEngineer}}
+      {{#link-to 'engineer' class="navbar-brand" }}{{#paper-button raised=true }}{{paper-icon 'home'}}{{/paper-button}}{{/link-to}}
```
Comment (spoonscen) [inline app/templates/components/my-application.hbs:10]: nest this like you have it below

File: app/styles/layout.scss [hunk 2]
```diff
@@ -51,7 +73,56 @@ md-toolbar .md-button {
...
+}
+
+.view-engineers-container md-card {
+  margin: 1px;
+}
+
+//  {
+//   color: black;
+//   border: 2px solid rgba(55, 71, 79, 0.36);
+// }
```
Comment (spoonscen) [inline app/styles/layout.scss:94]: delete unused code :)

File: app/styles/layout.scss [hunk 3]
```diff
@@ -51,7 +73,56 @@ md-toolbar .md-button {
...
+}
+
+
+.view-engineers-title-container, .message-container {
+  border-bottom: 5px solid #3f51b5;
+}
+
+.view-engineers-title-container h2 {
+  margin-left: 6px;
+}
```
Comment (spoonscen) [inline app/styles/layout.scss:116]: Sass allows for nesting. Lets use that.
```sass
.view-engineers-title-container {
  width: 100%;
  display: flex;
  flex-direction: row;
  align-items: center;
  height: 75px;
  
  h2 {
    margin-left: 6px;
  }
}
```
then you an combine this with declaration on line 101

File: app/styles/layout.scss [hunk 4]
```diff
@@ -51,7 +73,56 @@ md-toolbar .md-button {
...
+.view-engineers-title-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 75px;
+}
+
+
+.view-engineers-title-container, .message-container {
```
Comment (spoonscen) [inline app/styles/layout.scss:110]: if these two classes are the same, why not use the same class in the template?

File: app/templates/components/change-password-form.hbs [hunk 5]
```diff
@@ -1,16 +1,5 @@
...
-    {{#link-to 'engineer'}}
-      {{#paper-button fab=true}}Back{{/paper-button}}
-    {{/link-to}}
-  {{/if}}
 
-{{#if isRecruiter}}
-  {{#link-to 'recruiter'}}
-    {{#paper-button fab=true}}Back{{/paper-button}}
-  {{/link-to}}
-{{/if}}
```
Comment (spoonscen) [inline app/templates/components/change-password-form.hbs:3]: remove all the xtra whitespace here. Aim to keep your templates really clean.

File: app/templates/components/my-application.hbs [hunk 6]
```diff
@@ -1,59 +1,59 @@
...
+  <div class="paper-right">
+    {{#if isAuthenticated}}
+      {{#link-to "change-password"}}
+        {{#paper-button raised=true}}Change Password{{/paper-button}}{{/link-to}}
+      {{#paper-button raised=true}}
+        <div {{action "signOut"}}>Sign Out</div>
+      {{/paper-button}}
+
+    {{else}}
+      {{!-- inconsistent indentation fixes bug with link-to --}}
```
Comment (spoonscen) [inline app/templates/components/my-application.hbs:52]: Can you link to the bug? Is there an issue filed on github? If not, can you make an issue for ember paper and then link it here?

would separating these into two partials to make it consistent?

File: app/templates/components/recruiter-conversations.hbs [hunk 7]
```diff
@@ -8,5 +8,5 @@
 {{#if conversations}}
```
Comment (spoonscen) [inline app/templates/components/recruiter-conversations.hbs:8]: I think you can change this to 
```hbs
{{#if !conversations}}
  <h4>You haven't invited anyone to lunch yet or you deleted all the invitations from your inbox.</h4>
{{/if}
```
then you can leave off the `else` clause

Comment (spoonscen) [reply app/templates/components/recruiter-conversations.hbs:8]: Also clean up the spacing here in the lines above.

File: package.json [hunk 8]
```diff
@@ -22,47 +22,47 @@
...
-    "ember-resolver": "2.1.0",
-    "grunt": "1.0.1",
-    "grunt-concurrent": "2.3.1",
-    "grunt-contrib-jshint": "1.1.0",
-    "grunt-jscs": "3.0.1",
-    "grunt-jsonlint": "1.1.0",
-    "load-grunt-config": "0.19.2",
-    "loader.js": "4.1.0",
-    "time-grunt": "1.4.0"
+    "active-model-adapter": "^2.1.1",
```
Comment (spoonscen) [inline package.json:25]: did you have to bump all of these?, I'd always recommend hardcoding these. 
https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/

File: app/templates/components/my-application.hbs (hunk 6 above)
Comment (aprilsourz) [reply app/templates/components/my-application.hbs:52]: I filed an issue on the repo for ember-paper.

https://github.com/miguelcobain/ember-paper/issues/745

File: app/styles/layout.scss (hunk 4 above)
Comment (aprilsourz) [reply app/styles/layout.scss:110]: I agree, `.message-container` is more semantic in the template, but I can change the name of the class and use it both places.

File: app/styles/layout.scss (hunk 2 above)
Comment (aprilsourz) [reply app/styles/layout.scss:94]: Will do!

File: app/templates/components/recruiter-conversations.hbs (hunk 7 above)
Comment (aprilsourz) [reply app/templates/components/recruiter-conversations.hbs:8]: The app breaks and wont build with `{{#if !conversations}}`.
This works though.
```
{{#unless conversations}}
  <h4>You haven't invited anyone to lunch yet or you deleted all the invitations from your inbox.</h4>
{{/unless}}
```

File: package.json (hunk 8 above)
Comment (aprilsourz) [reply package.json:25]: When I took out the `^`I couldn't spin up my dev server. I got errors that I had different versions of some of the modules than specified. Is there another step I missed?

**Pull request closed without merging**